ci-operator/step-registry/upi/install/aws/cluster2/upi-install-aws-cluster2-commands.sh

```diff
@@ -0,0 +1,603 @@
+#!/bin/bash
```

Comment:
The reason of introducing this new file is for not touching the existing one - `upi-install-aws-cluster`, right?

Reply:
yes, both of install and destroy scripts can not be re-used.